client/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -52,7 +48,25 @@ const LoginForm: React.FC<Props> = (props: Props) => {
     }
   }
 
-  return (
+  return !loginLocal ? (
```

Comment:
can we use convention

`if (loginLocal) return <div...>`

later this convention of `?:` will cause readability issues